src/cli/add_completions.zig

```diff
@@ -66,7 +66,9 @@ pub const index: Index = if (Environment.isDebug) Index.initFill(&.{"OOMWorkArou
                 break;
             }
         }
-        array.set(@as(FirstLetter, @enumFromInt(i)), &record);
+
+        const cloned = record;
+        array.set(@as(FirstLetter, @enumFromInt(i)), &cloned);
```

Comment:
Ah this is new to me. It needs to be `const`?

Reply:
`var record: [count][]const u8 = undefined;`

is comptime mutable memory

`const cloned = record;` makes it constant